submissions/causal_machine_learning_in_healthcare.md

```diff
@@ -0,0 +1,420 @@
+(Lundberg and Lee 2017). While both of these approaches are
+model-agnostic, their sampling-based nature is computationally
+demanding. AME shows similar estimation accuracy for the feature
+importances with significantly lower computational requirements.
+Furthermore, the associations identified by the AME model with a
+properly tuned MGE\/MSE tradeoff were consistent with those reported by
+domain experts, which was not the case for the other evaluated models.
+
+However, there are some limitations to AME models. The model structure
+is fixed, which can result in worse predictive performance for certain
+tasks. Moreover, as the MSE\/MGE is jointly optimized, the MSE generally
+increases when more importance is given to the MGE, meaning there is a
+tradeoff between predictive performance and accurate importance
+estimation. Furthermore, the direction of the influence (positive or
+negative) is not inferred and with many features (and therefore experts,
+if a one-to-one mapping is used), the optimization can become
+intractable.
+
+## CXPlain Model
+
+CXPlain addresses the issue of the fixed model structure and increasing
+MSE that arises when using AME by training a separate explanation model
+and allowing arbitrary predictive models (Schwab and Karlen 2019). The
+explanation model treats the predictive models as blackboxes and
+calculates its outputs with and without each input feature. Note that a
+different strategy for obtaining the predictions without a feature is
+needed than in AME models as the predictive model now is arbitrary and
+cannot be modified. This can be accomplished by masking the feature with
+zeroes, replacing it with the mean value, or using more sophisticated
+masking schemes. Given these outputs, the (normalized) decrease in error
+is calculated for every input feature and the Kullback-Leibler
+divergence between this decrease and the models importance scores
+\(a_i\) is used as the objective function like in AME models. However,
+this objective function is now optimized individually, the task of
+producing feature importance estimates is therefore transformed into a
+supervised learning task with a Granger-causal objective function.
+
+Because some feature importance estimates may themselves be very
+unreliable (Zhang et al. 2019), CXPlain additionally provides
+uncertainty estimates for each feature importance estimate. It uses
+bootstrap resampling for that, i.e. training the explanation model on
+different subsets of the data (possibly containing duplicates) and using
+the importance scores of the runs to construct confidence intervals.
+
+As AME, CXPlain provided more accurate feature importance estimates than
+LIME and SHAP, while being model-agnostic and still computationally
+efficient. Even though the approach works with arbitrary models, the
+accuracy of the estimates does depend on the predictive model and some
+model architectures seem to be better suited for explanation models.
+
```

Comment:
A conclusion and future work section, even if it's short would be great to have here.